docs/buildah-bud.md

```diff
@@ -4,7 +4,9 @@
 buildah\-bud - Build an image using instructions from Dockerfiles.
 
 ## SYNOPSIS
-**buildah** **bud | build-using-dockerfile** [*options* [...]] **context**
+**buildah build-using-dockerfile** [*options*] *context*
```

Comment:
I would rather list both.
**buildah bud** [*options*] *context*

Reply:
> I would rather list both.
> **buildah bud** [*options*] *context*

How do you feel about [the belt-and-suspenders approach][1]?

[1]: https://github.com/projectatomic/buildah/blob/4a2b66a55ba513480c1dd7b35055c9f724f6d310/docs/buildah-bud.md#synopsis